packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -61,13 +64,22 @@ export async function runWithRetry<T>(
                 }, err);
                 throw err;
             }
+
+            if (progress.cancel?.aborted === true) {
+                throw new NonRetryableError(
+                    "runWithRetryAborted",
+                    OdspErrorType.fetchTimeout,
+                    { eventName: `runWithRetryAborted_${fetchCallName}`, driverVersion: pkgVersion },
```

Comment:
This eventName will never make it to the logs :)  Whenever this error is logged, the event will have an eventName, and it won't be overwritten by this prop.  Maybe we want to combine this case with the case above, or at least mimic it.

The symmetry makes sense because if the api checks for the abort signal (notice it gets passed `progress.cancel`) then it will probably handle that by throwing an error and end up in the first block, whereas if it doesn't then we'll handle it here.  So would be nice for the logging to be the same I think.